core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -44,6 +79,153 @@ func (l *logPollerWrapper) Ready() error {
 	return nil
 }
 
-func (l *logPollerWrapper) LatestRoutes() (activeCoordinator common.Address, proposedCoordinator common.Address, err error) {
-	return common.Address{}, common.Address{}, nil
+// "public" methods of LogPollerWrapper
+func (l *logPollerWrapper) LatestRequests() ([]evmRelayTypes.OracleRequest, error) {
+	// TODO poll both active and proposed coordinators for requests and parse them
+
+	// logs, err := lp.Logs(
```

Comment:
You should be able to unit-test this easily with a LogPoller mock. Also feel free to modify the Request struct if it makes parsing easier.